mirage/config.js

```diff
@@ -85,6 +85,21 @@ export default function() {
 
   this.delete('/user_categories/:id');
 
+  this.get('/skills');
+  this.get('/user_skills');
+
+  // TODO: Make this work when relationships work
```

Comment:
@begedin I don't think I can. I think Sam mentioned this was still many-to-many so it wouldn't work because Mirage doesn't support it out of the box right now.